### ISSUE TITLE
Allow aria-hidden=true on picture element

### ISSUE THREAD

**Issue by scottaohara** [w3c/html-aria#346]
presently the picture element allows for no roles or aria-* attributes, which generally makes sense because the picture element itself is not an accessible object.   That said, there could be a situation where a picture that contains a decorative image needs to be hidden..... and for whatever reason the expected `alt=""` on the fallback image cannot be used, but `aria-hidden=true` on the picture element could.   Upon testing this just now, seems like this would be a viable option as Chromium browsers, Firefox and Webkit all ignore the child `<img>` in this situation.

**Comment by stevefaulkner** [comment]
WFM
